python/lsst/analysis/tools/interfaces/_task.py

```diff
@@ -60,6 +60,10 @@
 from ._interfaces import KeyedData, PlotTypes
 from ._metricMeasurementBundle import MetricMeasurementBundle
 
+# TODO: This rcParams modification is a temporary solution, hiding
+# a matplotlib warning indicating too many figures have been opened.
+# When DM-39114 is implemented, this should be removed.
+plt.rcParams.update({"figure.max_open_warning": 0})
```

Comment:
This gets run exactly once at the import time and even if the individual tasks override the `runQuantum` method.